src/Data/ObfPoiSectionReader_P.cpp

```diff
@@ -1455,6 +1611,14 @@ void OsmAnd::ObfPoiSectionReader_P::readNameIndexDataAtom(
                         d = qAbs(xy31->x - position31.x) + qAbs(xy31->y - position31.y);
                     outDataOffsets.insert(dataOffset, d);
                 }
+                
+                QList<int> bboxResult;
+                section->bboxIndexCache.query(AreaI(position31.x, position31.y, position31.x, position31.y), bboxResult);
```

Comment:
`AreaI(position31.x, position31.y, position31.x, position31.y)` - is it correct?